addons/gen.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+WD=$(dirname "$0")
+WD=$(cd "$WD"; pwd)
+
+set -eux
+
+# This script sets up the plain text rendered deployments for addons
+# See samples/addons/README.md for more information
+
+TEMPLATES="${WD}/../templates"
+DASHBOARDS="${WD}/dashboards"
+TMP=$(mktemp -d)
+
+# Set up prometheus
+helm template prometheus prometheus \
+  --namespace "replace-me-namespace" \
+  --version 13.2.1 \
+  --repo https://prometheus-community.github.io/helm-charts \
+  -f "${WD}/values-prometheus.yaml" \
+  > "${TEMPLATES}/prometheus.yaml"
+
+sed -i'.orig' 's/replace-me-namespace/{{ .Release.Namespace }}/g' "${TEMPLATES}/prometheus.yaml"
+sed -i'.orig' '1i\
+{{- if .Values.prometheus.enabled }}
+' "${TEMPLATES}/prometheus.yaml"
+sed -i'.orig' -e '$a\
+{{- end }}' "${TEMPLATES}/prometheus.yaml"
+rm "${TEMPLATES}/prometheus.yaml.orig"
+
+function compressDashboard() {
+  < "${DASHBOARDS}/$1" jq -c  > "${TMP}/$1"
+}
+
+# Set up grafana
+{
+  helm template grafana grafana \
+    --namespace "replace-me-namespace" \
+    --version 6.2.1 \
+    --repo https://grafana.github.io/helm-charts \
+    -f "${WD}/values-grafana.yaml"
+
+  # Set up grafana dashboards. Compress to single line json to avoid Kubernetes size limits
+  compressDashboard "consul-server-monitoring.json"
```

Comment:
this is the only default dashboard I found. are there we would like to ship by default? also, this requires the stats port to be configured on Consul which i don't think is the default, so might need tinkering.

Reply:
assuming this is for demos, it might be good to keep an eye out, but can be punted to later. @codyde and @DerekStrickland, do y'all have frequently displayed dashboards for grafana?

Reply:
I don't, but @mkcp created [this one](https://grafana.com/grafana/dashboards/13396) that needs these changes in order to work.

Reply:
Yup! That dashboard is the only one included so far 😄